eng/common/scripts/Update-DocsMsToc.ps1

```diff
@@ -181,22 +181,25 @@ foreach ($service in $serviceNameList) {
   $clientPackages = $packagesForToc.Values.Where({ $_.ServiceName -eq $service -and ('client' -eq $_.Type) })
   $clientPackages = $clientPackages | Sort-Object -Property Package
   foreach ($clientPackage in $clientPackages) {
-    $packageItems += GetClientPackageNode -clientPackage $clientPackage
+    $packageItems += GetPackageNode -clientPackage $clientPackage
   }
 
   # All management packages go under a single `Management` header in the ToC
   $mgmtPackages = $packagesForToc.Values.Where({ $_.ServiceName -eq $service -and ('mgmt' -eq $_.Type) })
   $mgmtPackages = $mgmtPackages | Sort-Object -Property Package
-  if ($mgmtPackages) {
-    $children = &$GetDocsMsTocChildrenForManagementPackagesFn `
-      -packageMetadata $mgmtPackages `
-      -docRepoLocation $DocRepoLocation
-
+  $serviceReadmeBaseName = $service.ToLower().Replace(' ', '-').Replace('/', '-')
+  $mgmtItems = @()
+  foreach ($pkg in $mgmtPackages) {
+    # $children = &$GetDocsMsTocChildrenForManagementPackagesFn `
+    #   -packageMetadata $mgmtPackages `
+    #   -docRepoLocation $DocRepoLocation
+    $mgmtItems += GetPackageNode -clientPackage $pkg
+  }
+  if ($mgmtItems) {
     $packageItems += [PSCustomObject]@{
       name     = 'Management'
-      # There could be multiple packages, ensure this is treated as an array
-      # even if it is a single package
-      children = @($children)
+      href     = "~/docs-ref-services/{moniker}/$serviceReadmeBaseName.md"
```

Comment:
Does this insert the service overview into the ToC underneath management? 

* ...
* Key vault
  * Overview (service level overview, expected) 
  * Keys
    * Overview (package level overview) 
    * @azure/keyvault-keys (package ref docs) 
  * ...
  * Management
    * Overview <======== (**Is this re-inserting the service level overview?**) 
    * Key Vault Management (whatever the package DisplayName is)
      * Overview (package level overview) 
      * @azure/arm-keyvault (package ref docs)
* ...

Reply:
Good catch!!